### ISSUE TITLE
Water with DH not rendering when seen through glass

### ISSUE THREAD

**Issue by Zachar1er** [sixthsurge/photon#146]
### What happened?

When I watch an ocean through glass, the "normal" water looks normal, but the one from DH is all black. It doesn't happen with others shaders ( the second screenshot is with BSL shader )
![2024-04-21_18 31 18](https://github.com/sixthsurge/photon/assets/59703589/a583fde5-3d05-45a6-9477-0372033703e9)
![2024-04-21_18 26 15](https://github.com/sixthsurge/photon/assets/59703589/5b00e4cf-b96b-43e6-8eef-96d94ad9e4db)

### Minecraft Version

1.20.2

### Which shader mod are you using?

Iris

### Shader mod version

1.7.0 - 2.20.4 -  6bad887 - dirty

### Shader settings

```conf
#Sun Apr 21 18:30:46 CEST 2024
COLORED_LIGHTS=true
CUSTOM_SKY=true
HANDHELD_LIGHTING=true
SH_SKYLIGHT=true
MOON_I=0.80
VANILLA_MOON=false
WATER_CAUSTICS=true
AURORA_FREQUENCY=1.25
```

**Comment by Allexio** [comment]
1.20.2 isn't supported by the latest iris dev versions though, is it?

**Comment by Zachar1er** [comment]
Yes, I'm in 1.20.4, I put the wrong version when reporting the bug sorry
